update to use from_s3_cache by default

matchings opened the pull request.

alexpiet commented on visual_behavior/data_access/loading.py:
```diff
@@ -190,8 +217,8 @@ def get_filtered_ophys_experiment_table(include_failed_data=False, release_data_
-        cache = bpc.from_lims(data_release_date='2021-03-25')
-        # cache = get_visual_behavior_cache(from_s3=False, release_data_only=True)
+        cache = bpc.from_lims(data_release_date=['2021-03-25', '2021-08-12'])
+        print('use get_released_ophys_experiment_table to get August release data in addition to March release')
```
The comment seems to be out-of-date with the line above (220)

matchings replied:
thanks, will change

alexpiet reviewed (commented):
Before you merge, can you document the best way to use these functions? Everything in VBA keeps changing, and I struggle to keep up

alexpiet commented:
Before merging @matchings can you merge this PR into `update_nwb_loading`? 

https://github.com/AllenInstitute/visual_behavior_analysis/pull/786

alexpiet commented:
@matchings Something broke `loading.get_filtered_ophys_experiment_table()` I'm not sure if its this PR, or something else.

`experiment_table = loading.get_filtered_ophys_experiment_table()` now crashes

alexpiet commented:
@matchings 
`loading.get_platform_paper_experiment_table()` throws a bunch of warnings about trying to set on a copy of a slice.

alexpiet reviewed (changes requested):
See three issues in the comments

farznaj commented on visual_behavior/data_access/loading.py:
```diff
@@ -185,7 +185,12 @@ def get_platform_paper_experiment_table():
+    experiment_table = utilities.add_n_relative_to_novel_1(experiment_table)
+    experiment_table = utilities.add_last_familiar(experiment_table)
+    experiment_table = utilities.add_last_familiar_active(experiment_table)
+    experiment_table = utilities.add_first_novel(experiment_table)
```
lets change the column first_novel to first_post_novel.

matchings replied:
how about 'first_novel_repeat' or 'second_novel' instead?

matchings replied:
or even more explicit, "first_passing_novel_repeat" or "second_passing_novel", just so we don't confuse absolute order (the true experience of the mouse) with the order of sessions that are available for analysis (only the passing ones)

matchings commented:
@alexpiet I believe i have addressed the issues you brought up above, however i want to point out that you should not be using` loading.get_filtered_ophys_experiment_table()` for platform paper analysis anymore, as the default behavior of this function is to get data from lims. 

The new function` loading.get_platform_paper_experiment_table()` gets the final dataset from a set cache directory that contains the NWB files downloaded from AWS. This is what we should all be using going forward. @yavorska-iryna  @farznaj
